File: Python/String-1/non_start.py
"""

Given 2 strings, return their concatenation, except omit the first char of each. The strings will be at least length 1.


non_start('Hello', 'There') → 'ellohere'
non_start('java', 'code') → 'avaode'
non_start('shotl', 'java') → 'hotlava'

@author unobatbayar

"""
def non_start(a, b):
  first = a[1:]
  last = b[1:]
  return first + last

File: Java/String-1/nonStart.java
/**
 * Given 2 strings, return their concatenation, except omit the first char of each. The strings will be at least length 1.

nonStart("Hello", "There") → "ellohere"
nonStart("java", "code") → "avaode"
nonStart("shotl", "java") → "hotlava"
@author unobatbayar
 */

public String nonStart(String a, String b) {
    String ac = a.substring(1, a.length());
    String bc = b.substring(1, b.length());
    return ac + bc;
  }
  